code/datums/mutations/hulk.dm

```diff
@@ -51,6 +52,7 @@
 	REMOVE_TRAIT(owner, TRAIT_PUSHIMMUNE, TRAIT_HULK)
 	REMOVE_TRAIT(owner, TRAIT_CHUNKYFINGERS, TRAIT_HULK)
 	REMOVE_TRAIT(owner, TRAIT_IGNOREDAMAGESLOWDOWN, TRAIT_HULK)
+	REMOVE_TRAIT(owner, TRAIT_HULK, TRAIT_HULK)
```

Comment:
we have if(dna && dna.check_mutation(HULK))

Reply:
Last time I used that check several people told me to check traits instead so I made it a trait so I could do this aaaaa